From bf0d02e501af3616ec4f4f7a20668b45358a76fe Mon Sep 17 00:00:00 2001
From: Nitin Garg <nitin.garg@nxp.com>
Date: Wed, 27 Jun 2018 13:40:03 -0500
Subject: [PATCH 4109/5242] MLK-18090: 8QM: Revert: Fix the rpmsg addresses

commit  3437c6afdf7b789373653eac10fd04ba75aa04db from
https://source.codeaurora.org/external/imx/linux-imx.git

This reverts commit e42b28ed21b
The address change isn't needed for iMX8QM arm2
boards anymore.
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8q-arm2.dtsi |    6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8q-arm2.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8q-arm2.dtsi
index 85c5b8c..b647e40 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8q-arm2.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8q-arm2.dtsi
@@ -1014,10 +1014,9 @@
 &rpmsg{
 	/*
 	 * 64K for one rpmsg instance:
-	 * --0xb8000000~0xb800ffff: pingpong
 	 */
 	vdev-nums = <1>;
-	reg = <0x0 0xb8000000 0x0 0x10000>;
+	reg = <0x0 0x90000000 0x0 0x10000>;
 	status = "okay";
 };
 
@@ -1028,10 +1027,9 @@
 &rpmsg1{
 	/*
 	 * 64K for one rpmsg instance:
-	 * --0xb8100000~0xb810ffff: pingpong
 	 */
 	vdev-nums = <1>;
-	reg = <0x0 0xb8100000 0x0 0x10000>;
+	reg = <0x0 0x90100000 0x0 0x10000>;
 	status = "okay";
 };
 
-- 
1.7.9.5

